File: app/HateWait/src/main/java/HateWait/vo/QueueListSerializable.java
package HateWait.vo;

import java.io.Serializable;
import java.util.List;

public class QueueListSerializable implements Serializable {
	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;
	public static final String INFO="STRQUE";
	public int autonum;
	public List<QueueInfoVo> qivo;
	@Override
	public String toString() {
		return "QueueListSerializable [ INFO=" + INFO + ", autonum=" + autonum + ", qivo=" + qivo + "]";
	}
	public static String getInfo() {
		return INFO;
	}
	public int getAutonum() {
		return autonum;
	}
	public void setAutonum(int autonum) {
		this.autonum = autonum;
	}
	public List<QueueInfoVo> getQivo() {
		return qivo;
	}
	public void setQivo(List<QueueInfoVo> qivo) {
		this.qivo = qivo;
	}
}

File: app/src/main/java/com/example/hatewait/serialize/QueueListSerializable.kt
package com.example.hatewait.serialize

import java.io.Serializable

class QueueListSerializable : Serializable {
    var autonum = 0
    var qivo: List<QueueInfoVo>? = null
    override fun toString(): String {
        return "QueueListSerializable [ INFO=" + info + ", autonum=" + autonum + ", qivo=" + qivo + "]"
    }

    companion object {
        private const val serialVersionUID = 1L
        const val info = "STRQUE"
    }
}